packages/lesswrong/components/tagging/SubforumMembersDialog.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useMulti } from '../../lib/crud/withMulti';
+import DialogContent from '@material-ui/core/DialogContent';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  titleRow: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    columnGap: 14,
+    padding: '0 24px',
+  },
+  title: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 20,
+    lineHeight: '26px',
+    fontWeight: 400,
+    textTransform: 'capitalize'
+  },
+  joinBtn: {
+    '& button': {
+      minHeight: 0,
+      fontSize: 12,
+      padding: 6
+    }
+  },
+  user: {
+    marginBottom: 20
+  }
+})
+
+const SubforumMembersDialog = ({classes, onClose, tag}: {
+  classes: ClassesType,
+  onClose: () => void,
+  tag: TagBasicInfo,
+}) => {
+  const { results: members, loading } = useMulti({
+    terms: {view: 'tagCommunityMembers', profileTagId: tag?._id, limit: 100},
+    collectionName: 'Users',
+    fragmentName: 'UsersProfile',
+    skip: !tag
```

Comment:
If `tag` is optional, should it be `tag?: TagBasicInfo` in the props type?

Reply:
But from looking at [where it's called](https://github.com/ForumMagnum/ForumMagnum/pull/5950/files#diff-12a1a8d9614617d2b288711faac74c020768ad91af7dbb165fe4162ef4695e7fR132) it seems that we're checking to make sure that a tag exists before opening the dialog, so I don't think `skip` is ever `true` (and we can remove the null-safe access on `tag?._id` above too)

Reply:
Oh I think this line is leftover from a previous version - thanks, I'll remove it (and check the rest of the file, since `tag` should not be optional)

Reply:
Actually, isn't this the same query as the one in `TagSubforumPage`, just with a `limit: 100`?  Even if apollo will query the cache and avoid a redundant DB query, maybe it makes sense to pass in the list of members as a prop?

Reply:
I was doing that originally, but I wanted the members list to properly update when you join or leave, which doesn't happen if it gets passed into `openDialog`. Seemed reasonable to get the list in the dialog instead. On the page itself I just wanted to get the member count, so if there's a better way to do that I'm happy to change it.